Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -140,6 +150,14 @@ open class Player: BaseObject {
     @objc open func setFullscreen(_ fullscreen: Bool) {
         core?.setFullscreen(fullscreen)
     }
+    
+    open var options: Options? {
+        return core?.options
+    }
+    
+    open var corePlugins: [UICorePlugin]? {
```

Comment:
After talking with some people, I realised we don't want to expose the plugin list. Instead, we could create a method that returns the instance of the plugin if the name is provided. Web player has it implemented here: https://github.com/clappr/clappr/blob/dev/src/components/player.js#L576

Reply:
It's done.
We implemented the method `getPlugin`, if you wanna take a look... 😜